### ISSUE TITLE
Confused by your instructions for OCR

### ISSUE THREAD

**Issue by ovizii** [Stirling-Tools/Stirling-PDF#821]
I am referring to this page here: https://github.com/Stirling-Tools/Stirling-PDF/blob/main/HowToUseOCR.md 

These instructions seem to contradict each other, could someone please clarify?

1. 
```
My OCR used to work and now doesn't!
Please update your tesseract docker volume path version from 4.00 to 5
```

So I though, OK, I'll change my previous volume:  `- ./trainingData:/usr/share/tesseract-ocr/4.00/tessdata` to  `- ./trainingData:/usr/share/tesseract-ocr/5/tessdata` 

but then I found the second instruction further down:

2.
Modify your docker-compose.yml file to include the following volume configuration:
```
services:
  your_service_name:
    image: your_docker_image_name
    volumes:
      - /location/of/trainingData:/usr/share/tessdata
```

Could somebody please clarify? I can't combine 1 + 2 as 2 does not contain any 4.00 or 5 version number.

Additional question:
Am I limited to a single language for OCR? Even though I downloaded a German training data file, when trying to OCR only English is offered.

**Comment by Frooodle** [comment]
You are using https://github.com/Stirling-Tools/Stirling-PDF/tree/v0.20.2
please refer to those instructions 
current details are for upcoming 0.21.0 release

although
``Please update your tesseract docker volume path version from 4.00 to 5`` comment should be removed now, thanks for that callout

**Comment by ovizii** [comment]
In case someone else finds this thread, here is the right set of instructions: https://github.com/Stirling-Tools/Stirling-PDF/blob/v0.20.2/HowToUseOCR.md

**Comment by Frooodle** [comment]
Closing as released new 0.22.0 version
